src/main/java/org/gridsuite/network/map/dto/BusbarSectionFinderTraverser.java

```diff
@@ -0,0 +1,47 @@
+/**
```

Comment:
This class is not a DTO : it should be in another package

Reply:
Same remark as in my previous response

Reply:
Done